index.html

```diff
@@ -36,6 +36,136 @@ <h1 class="heading-primary">I’m Jubaer. <span class="heading-primary__sapan">G
+            </div>
+        </section>
+
+        <section class="section-about-me">
+            <div class="container">
+                <h2 class="heading-primary">About Myself</h2>
+                <p class="section-hero__paragraph">Hello I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+                <a href="#" class="section-hero__link">LET’S CONNECT</a>
+                <div class="section-hero__social-media">
+                    <img src="./images/twiter.svg" alt="twiter logo" class="twiter-logo">
+                    <img src="./images/in.svg" alt="linked-in logo">
+                    <img src="./images/m.svg" alt="m logo">
+                    <img src="./images/victory.svg" alt="victory logo">
+                    <img src="./images/github.svg" alt="github logo">
+                </div>
+                <a href="#" class="card-cta">Get my resume</a>
+                <div class="about-me-tech">
+                    <div class="language">
+                        <div class="language-des">
+                            <h3 class="language-title">Languages</h3>
+                            <img src="./images/arrow-down-svgrepo-com.svg" alt="logo" class="arrow-logo">
+                        </div>
+                        <div class="language__details">
+                            <div class="language__details-container">
+                                <img src="images/javascript-logo.png" alt="logo" class="language__logo">
+                                <h4 class="language__title">JavaScript</h4>
+                            </div>
+                            <div class="language__details-container">
+                                <img src="./images/html-logo.png" alt="logo" class="language__logo">
+                                <h4 class="language__title">HTML</h4>
+                            </div>
+                            <div class="language__details-container">
+                                <img src="./images/css-logo.png" alt="logo" class="language__logo">
+                                <h4 class="language__title">CSS</h4>
+                            </div>
+                        </div>
+
+                        <div class="language-des">
+                            <h3 class="language-title">Frameworks</h3>
+                            <img src="./images/arrow-right-svgrepo-com.svg" alt="logo" class="arrow-logo">
+                        </div>
+                        <hr>
+                        <div class="language-des">
+                            <h3 class="language-title">Skills</h3>
+                            <img src="./images/arrow-right-svgrepo-com.svg" alt="logo" class="arrow-logo">
+                        </div>
+                        <hr>
+                    </div>
+                </div>
+            </div>
```

Comment:
- To help improve the semantics of your site, I suggest wrapping this section in a `ul` tag and wrap each corresponding text i.e `Languages, Frameworks, Skills` in a corresponding `li` tag. Also, wrap the technologies in a `ul` tag and wrap each of the technology in a corresponding `li` tag. Please fix this 🙏 . You might find the text below useful
```
<ul>
  <li>
	<div>
	    <p>Languages</p>
	    <img>
	</div>
	<ul>
	    <li>
		<img>
		<p>Javascript</p>
	    <li>
	    <li>
		<img>
		<p>HTML</p>
	    <li>
            <li>
		<img>
		<p>CSS</p>
	    <li>
	</ul>
  </li>
  <li>
     <p>FrameWorks</p>
     <img>
  </li>	
  <li>
     <p>Skills</p>
     <img>
  </li>
</ul>```